### ISSUE TITLE
Speech recognition software not showing item numbers for radio and checkbox

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2952]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Users with speech recognition software are not seeing the numbers for radio and checkboxes.

![image](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/425ad301-a491-482f-a679-48c582c2b8a1)

Video showing this with Mac Voice Control:

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/fc58345c-48bd-4aa3-9c3e-e8dd91555982

## What I expected to happen

Item numbers show beside the radio and checkbox inputs.

## Reproducing

- Formation version: N/A
- Device: (e.g. iPhone 8, Macbook) Mac
- Browser: (e.g. Firefox, IE 11) Chrome

Steps to reproduce:

1. Open **System Settings** > **Voice Control** - enable it and change **Overlay**: Item numbers
2. Open docs pages for va-radio and va-checkbox, there will be no visible item numbers.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [x] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

## Details

It appears that the browser's provided radio/checkbox may be getting pushed off-screen in order to provide a custom styled version in its place.

Example:
```
input[type=radio].usa-radio__input {
    margin-left: unset;
    opacity: unset;
    right: auto;
    left: -999em;
}

va-radio-option label.usa-radio__label::before {
    -webkit-box-shadow: #1b1b1b 0px 0px 0px 2px;
    box-shadow: #1b1b1b 0px 0px 0px 2px;
    height: 20px;
    width: 20px;
    margin-left: 2px;
}
```

This is typically done because historically browsers did not provide good options for styling a radio or checkbox. However, there may be other ways to do this, see [Custom Styling Radio Buttons: The Modern Way](https://dev.to/adbutterfield/custom-styling-radio-buttons-the-modern-way-the-butterfield-way-emk).

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2952?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rmessina1010 @rsmithadhoc

**Comment by Andrew565** [comment]
Confirmed that the `left: -999em` style rule used by both checkboxes and radio buttons is the culprit for Voice Control not finding these elements and making them "clickable". As a bonus, I also confirmed that this afflicts the USWDS components.

This will likely require re-making these components to leverage better modern styling techniques that do not require "replacing" the inputs with divs and spans. I will create an alternative and present it to all when ready.
